pkg/resto/resto.go

```diff
@@ -281,6 +281,9 @@ func PushConfigMulti(ctx context.Context, payload map[string]string, repoTag str
 	if _, ok := err.(unsupportedMediaType); ok {
 		return pushConfigLegacy(ctx, payload, pr, repo, labels)
 	}
+	if err != nil {
+		err = fmt.Errorf("Unable to push to: %s/%s\n%s", pr.domain, pr.path, err)
+	}
 	return digest, err
```

Comment:
Could also use `github.com/pkg/errors` :

```go
return digest, errors.Wrapf(err, "Unable to push to: %s/%s", pr.domain, pr.path)
```

Reply:
good idea. i did a force push with your suggestion.

another question i had related to this: Should the "namespace" attribute in files like:
 https://github.com/docker/app/blob/master/examples/hello-world/hello-world.dockerapp 

be removed? And if so, should the "name" attribute be updated to indicate it *needs* the namespace?

Reply:
@doanac you don't need to check for `err != nil` before wrapping the error, so you don't need that if (if the error is nil, the wrapped error will be nil too)